apps/core-app/src/renderer/src/modules/box/adapter/hooks/useClipboard.ts

```
@@ -4,10 +4,14 @@ import { touchChannel } from '~/modules/channel/channel-core'
 
-// Utility function to normalize various timestamp formats
+const AUTOFILL_SHORT_TEXT_LIMIT = 80
+const AUTOFILL_TIMESTAMP_TTL = 60 * 60 * 1000
+const AUTOFILL_CLEANUP_PROBABILITY = 0.1
+const AUTOFILL_LOG_PREVIEW_LENGTH = 30
+
+// Normalize timestamp inputs so autopaste doesn't choke on weird types
 function normalizeTimestamp(value?: string | number | Date | null): number | null {
-  if (value === null || value === undefined)
-    return null
+  if (value === null || value === undefined) return null
```

ESLint

[error] Expect newline after if

(antfu/if-newline)